modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/supplemental_claims_controller.rb

```diff
@@ -43,25 +43,13 @@ def validate
   end
 
   def schema
-    # TODO: Return full schema after we've validated all Non-Veteran Claimant functionality
     response = AppealsApi::JsonSchemaToSwaggerConverter.remove_comments(
       AppealsApi::FormSchemas.new(
         SCHEMA_ERROR_TYPE,
         schema_version: 'v2'
       ).schema(self.class::FORM_NUMBER)
     )
 
-    if Settings.modules_appeals_api.documentation.wip_docs&.include?(:sc_v2_claimant)
```

Comment:
There are some places that look for either [nod_v2_claimant](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/support/rswag_config.rb#L407-L411) and [sc_v2_claimant](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/support/rswag_config.rb#L538-L547) in our specs that we can remove as well. Should probably just grep around for those setting keys.

Another spot to double check is the NODv2 controller - it's probably removing claimant details in its `schema` action, too.

Reply:
On the note of having multiple places where we take in the schema and modify it (both in the controllers and in the docs) - I'm wondering if we shouldn't write some presenter to wrap the data up & get the modified schema from. That way there'd only be one, consistent place that the data manipulation happens 🤔 It's a bit unnecessary right now, since we're no longer manipulating the schema, but likely something we'll be doing in the near future regardless...

Reply:
I had added a [small thing for claims](https://github.com/department-of-veterans-affairs/vets-api/blob/master/rakelib/rswag.rake#L40-L53) to strip out some deprecated OAS v3 values post-rswag swagger generation, but that's not quite the same thing as this direct modification of the schema. Agreed it'd be something to keep an eye out for in future changes